src/mem/epoch/garbage.rs

```diff
@@ -128,17 +128,9 @@ impl ConcBag {
         self.head.store(ptr::null_mut(), Relaxed);
 
         while head != ptr::null_mut() {
-            let mut n = from_raw(head);
+            let mut n = Box::from_raw(head);
             n.data.collect();
             head = n.next.load(Relaxed);
         }
     }
 }
-
-fn into_raw<T>(b: Box<T>) -> *mut T {
```

Comment:
This particular pattern was very useful. It made it very obvious what needed to be changed :+1: